site/app/libraries/database/DatabaseQueries.php

```diff
@@ -3452,7 +3452,11 @@ public function getSeekMessageByUserId($g_id, $user_id) {
             [$g_id, $user_id]
         );
 
-        return $this->course_db->rows()[0]['message'];
+        $rows = $this->course_db->rows();
+        if (empty($rows)) {
```

Comment:
```suggestion
        if (count($rows) === 0) {
```

In general, it is preferable to explicitly count the number of rows instead of using `empty()` since `empty()` can come with some unintended results.  See [this](https://www.php.net/manual/en/function.empty.php) documentation page for further details.  It probably wouldn't make a huge difference here, but it's just one more place we'll have to clean up whenever I finish some of my linting PRs.